app/views/layouts/portal.html.erb

```diff
@@ -40,6 +40,11 @@ By default, it renders:
       </main>
     </div>
   </body>
+  <style>
+  :root {
+    color-scheme: '<%= @theme %>';
```

Comment:
Grabs the `@theme` using a `<style>` tag is the best practice to solve this?

Reply:
@jsobralgitpush Thanks for pointing this out. However, we need to set the color-scheme in order to enable dark mode for native browser components such as scrollbars, inputs, select, etc. The value of `@theme` can be either `dark` or `light`. I hope this clarifies the situation. Please let me know if you have a better solution to address this.